build/commands/lib/buildChromiumRelease.js

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// A script to build and pack a Chromium release build from scratch
+// Reuses the same /src folder
+// Designed to be used on CI, but should work locally too.
+// The script includes syncing; there is no need to run npm run sync before.
+
+const config = require('./config')
+const util = require('./util')
+const path = require('path')
+const fs = require('fs-extra')
+const syncUtil = require('./syncUtils')
+const Log = require('./logging')
+
+// Use the same filename as for Brave archive.
+const getOutputFilename = () => {
+  const platform = (() => {
+    if (config.getTargetOS() === 'win')
+      return 'win32'
+    if (config.getTargetOS() === 'mac')
```

Comment:
We have to use `target_arch` here, because we have two different mac: arm64 and x64.